compile/ivy/src/main/scala/sbt/compiler/ComponentCompiler.scala

```diff
@@ -14,6 +15,7 @@ object ComponentCompiler {
   val compilerInterfaceSrcID = compilerInterfaceID + srcExtension
   val javaVersion = System.getProperty("java.class.version")
 
+  @deprecated("Please specify `ivyConfiguration`.", "0.13.10")
```

Comment:
Avoid "please" and "thank you"s. The imperative sentence is normally fine e.g. "press any keys."